docs/_ug/TryingFirstCommand.md

```diff
@@ -13,7 +13,7 @@ One of the available commands in FoodRem is the command to create a new item.
 The first word of every command allows FoodRem to distinguish different commands.
 
 * `new` tells FoodRem that this is the command to create a new item
-* [Flags](#flags) such as `n/` and `qty/` are delimiters that enable FoodRem to distinguish different parameters supplied by you without ambiguity
+* [Flags](#flags) such as `n/` and `qty/` are [[ delimiter:delimiters]] that enable FoodRem to distinguish different parameters supplied by you without ambiguity
```

Comment:
Formatting. Note that there does not exist a glossary entry called delimiter, so this will remain as `[[ delimiter:delimiters ]]` and not actually get transformed in to an autoglossary link. Having said that, it's fine, as my PR has this change to fix it.

```suggestion
* [Flags](#flags) such as `n/` and `qty/` are [[ delimiter:delimiters ]] that enable FoodRem to distinguish different parameters supplied by you without ambiguity
```

Reply:
@Eugenetanwl3881 don't worry about this change, I've fixed it on my PR. Just resolve this conversation to acknowledge, thanks!

Reply:
Ok! Thanks. Apologies for the formatting mistakes (I wasn't aware about the extra spaces needed and unecessary aliases 😅 ). Will take note of it when I do up the DG auto glossary!